### PR TITLE
Adding test for model in image end to end use case

### PR DESCRIPTION
**Pull request by vanajamukkara**
Adding test for Mii end to end use case :
create unique namespace for operator
create unique namespace for domain 
create service account for operator
install operator
verify operator pod is running
create image using model file
create domain using model in image
verify domain is created
verify admin and managed pods are started and ready
verify admin and managed services are running
delete domain 
delete domain image
uninstall operator
delete service account
delete domain namespace
delete operator namespace

Adding Docker Primitive class.

### CODE REVIEW

File: new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItMiiDomain.java [hunk 1]
```diff
@@ -0,0 +1,544 @@
...
+    logger.info("Created a new namespace called {0}", opNamespace);
+
+    logger.info("Creating unique namespace for Domain");
+    domainNamespace = createNamespace();
+    logger.info("Created a new namespace called {0}", domainNamespace);
+
+    // Create a service account for the unique opNamespace
+    logger.info("Creating service account");
+    String serviceAccountName = createSA(opNamespace);
+    logger.info("Created service account: " + serviceAccountName);
```
Comment (ddsharpe) [inline new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItMiiDomain.java:127]: logger.info("Created service account: {0}", serviceAccountName)

File: new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItMiiDomain.java [hunk 2]
```diff
@@ -0,0 +1,544 @@
...
+
+    // check operator is running
+    logger.info("Check Operator pod is running in namespace {0}", opNamespace);
+    waitForCondition(operatorIsRunning(opNamespace));
+
+  }
+
+  @BeforeEach
+  public void init() {
+    // create unique namespace for domain and do operator upgrade?
```
Comment (ddsharpe) [inline new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItMiiDomain.java:166]: If this was left empty intentionally, why?  Why not delete the init method?

File: new-integration-tests/src/test/java/oracle/weblogic/kubernetes/actions/TestActions.java [hunk 3]
```diff
@@ -222,6 +223,7 @@ public static boolean deleteNamespace(String namespace) throws ApiException {
 
   // ------------------------ Docker image  -------------------------
 
+
```
Comment (ddsharpe) [inline new-integration-tests/src/test/java/oracle/weblogic/kubernetes/actions/TestActions.java:226]: Two blank lines?  All of your other breaks like this only have one blank line before and after.

File: new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItMiiDomain.java (hunk 1 above)
Comment (vanajamukkara) [reply new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItMiiDomain.java]: changed

File: new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItMiiDomain.java (hunk 2 above)
Comment (vanajamukkara) [reply new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItMiiDomain.java]: Removed. One more test can be added for Operator upgrade.

File: new-integration-tests/src/test/java/oracle/weblogic/kubernetes/actions/TestActions.java (hunk 3 above)
Comment (vanajamukkara) [reply new-integration-tests/src/test/java/oracle/weblogic/kubernetes/actions/TestActions.java]: fixed

**Comment by vanajamukkara** [comment]
Closing this PR as I will be creating another PR against develop branch.

**Pull request closed without merging**